src/late_multicellular_stage/editor/MetaballEditorComponentBase.cs

```diff
@@ -69,7 +69,8 @@ public partial class MetaballEditorComponentBase<TEditor, TCombinedAction, TActi
 
     protected List<TMetaball> hoverMetaballData = new();
 
-    protected IMetaballDisplayer<TMetaball>? alreadyPlacedVisuals;
+    protected IMetaballDisplayer<TMetaball>? structuralMetaballDisplayer;
+    protected IMetaballDisplayer<TMetaball>? visualMetaballDisplayer;
```

Comment:
Could you add comments here clarifying the different purposes of these? I couldn't immediately from this code see what the reason was for splitting this into two.